src/language.rs

```diff
@@ -30,161 +30,148 @@ pub enum SupportedLanguageName {
     Lua,
 }
 
-impl SupportedLanguageName {
-    pub fn get_language(&self) -> SupportedLanguage {
-        *ALL_SUPPORTED_LANGUAGES_BY_SUPPORTED_LANGUAGE_NAME
-            .get(self)
-            .unwrap()
-    }
-}
+pub type BySupportedLanguage<T> = [T; 22];
```

Comment:
Use fixed-size arrays as the actual data structure for the "all supported languages" collections